docs/content/dagster-cloud/managing-deployments/alerts/email.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: "Email & Dagster Cloud | Dagster Docs"
+description: "Create alerts to send emails for specified Dagster Cloud events."
+
+platform_type: "cloud"
+feature: "alerting"
+---
+
+# Email & Dagster Cloud
+
+<ApplicableDagsterProduct />
+
+Dagster Cloud supports [sending alerts](/dagster-cloud/managing-deployments/alerts) as emails to one or more recipients.
+
+---
+
+## Creating an alert policy
+
+Creating an alert policy can be done using the Dagster Cloud UI or the `dagster-cloud` CLI and a YAML file.
+
+<TabGroup>
+<TabItem name="In the UI">
+
+### In the UI
+
+1. Create and configure an alert policy. Refer to the [alert policy documentation](/dagster-cloud/managing-deployments/alerts#creating-alert-policies) for more info.
+
+2. In the **Notification Service** section of the **Create alert policy** window, select the **Email** option.
+
+3. In the field that displays, enter the email address notifications should be sent to. To add multiple recipients, press **Enter** after each email address.
+
+   <!-- ![Email alert configured to alert two recipients](/images/dagster-cloud/alerts/cloud-email-alert-creation.png) -->
+
+   <Image
+   alt="Email alert configured to alert two recipients"
+   src="/images/dagster-cloud/alerts/cloud-email-alert-creation.png"
+   width={1571}
+   height={178}
+   />
+
+4. Click **Save policy**.
+
+</TabItem>
+<TabItem name="dagster-cloud CLI">
+
+### dagster-cloud CLI
+
+To define an email alert policy in code, use the `notification_service.email.email_addresses` key to provide a list of recipient email addresses. For example:
+
+```yaml file=/dagster_cloud/alerts/email_policy.yaml
+# alert_policies.yaml
+
+alert_policies:
+  - name: "email-alert-policy"
+    description: "An alert policy to email company executives during job failure."
+    tags:
+      - key: "level"
+        value: "critical"
+    event_types:
+      - "JOB_FAILURE"
+    notification_service:
+      email:
+        email_addresses:
+          - "richard.hendricks@hooli.com"
+          - "nelson.bighetti@hooli.com"
+```
+
+Then, use the `dagster-cloud` CLI to set the alert policies for the deployment:
+
+```bash file=/dagster_cloud/alerts/cli_commands.yaml startafter=start_set_alerts endbefore=end_set_alerts
+dagster-cloud deployment alert-policies sync -a /path/to/alert_policies.yaml
```

Comment:
we give this step for email but not for teams or slack

Reply:
Yep! I was waiting to see if you were good with this approach before doing it for the other guides. I'll add that in in my next couple of commits.